Scan environment:
  versions    - semgrep <sanitized semgrep_version> on python <sanitized_version>
  environment - running in environment github-actions, triggering event is pull_request
  semgrep.dev - authenticated as org_name


Traceback (most recent call last):
  File "/home/runner/work/semgrep/semgrep/semgrep/semgrep/commands/wrapper.py", line 35, in wrapper
    func(*args, **kwargs)
  File "/home/runner/work/semgrep/semgrep/semgrep/semgrep/commands/ci.py", line 317, in ci
    with fix_head_if_github_action(metadata):
  File "/usr/lib/python3.8/contextlib.py", line 113, in __enter__
    return next(self.gen)
  File "/home/runner/work/semgrep/semgrep/semgrep/semgrep/commands/ci.py", line 83, in fix_head_if_github_action
    assert metadata.head_ref is not None  # Not none when github action PR
AssertionError
